dspace/config/modules/rest.cfg

```diff
@@ -39,8 +39,11 @@ rest.properties.exposed = orcid.application-client-id
 rest.properties.exposed = orcid.authorize-url
 rest.properties.exposed = orcid.scope
 rest.properties.exposed = orcid.disconnection.allowed-users
-rest.properties.exposed = websvc.opensearch.svccontext
-rest.properties.exposed = submit.type-bind.field
+rest.properties.exposed = registration.verification.enabled
+rest.properties.exposed = google.recaptcha.key.site
+rest.properties.exposed = google.recaptcha.key.secret
```

Comment:
I don't believe the `secret` should ever leave the backend.  Otherwise, we have a possible security issue, as we'd be publicly sharing both the `site` and the `secret`.

From looking at the frontend code, it doesn't appear the `secret` is needed there anyways.

Reply:
Again the `google.recaptcha.key.secret` should be removed from this list. I don't think it should be shared.